tests/packages/recent/requirements.txt

```diff
@@ -0,0 +1 @@
+robotframework==3v4.1.2rc2
```

Comment:
What version is it?

Reply:
this was only for test purposes, just to check if dependabot would start a PR to the latest version
https://github.com/MalikMlitat/robotframework-robocop/pulls

Reply:
replaces now with latest (4.1.2)